test/e2e/framework/pod/resource.go

```diff
@@ -0,0 +1,689 @@
+/*
```

Comment:
For a follow-up PR, this package can probably use some clean-up.

Reply:
Should I create an issue and link it to #77095 ?

Reply:
Don't think an issue is needed, just a follow-up PR as part of this effort is fine I think :)

Reply:
sweet, will make a note of it then.